apps/web/lib/hooks/useSlots.ts

```diff
@@ -27,6 +27,7 @@ type UseSlotsProps = {
   eventLength: number;
   eventTypeId: number;
   minimumBookingNotice?: number;
+  minimumBookingNoticeType?: string | undefined;
```

Comment:
instead of string, maybe 

"minute" | "hour" | "day" ?

Reply:
I kept it as string so that if we wanted to add another value such as `business day` we could add that